API/projectClass/domainContrastsClass.m

```diff
@@ -12,7 +12,7 @@
     
     methods(Static)
```

Comment:
Hmm, there's a similar method in the parameters class as well. I can't see a static use either, so let's see if we can rewrite them.

Reply:
I think we've had them as static because we don't use the object in the methods. However, with the main contrasts class, we do pass in the domainsCalc and OilWaterCalc flags. Rewriting these methods would probably make them neater for that reason.